analysis/wwz/run_wwz4l.py

```diff
@@ -320,7 +320,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
     tstart = time.time()
 
     if executor == "futures":
-        exec_instance = processor.FuturesExecutor(workers=nworkers)
+        exec_instance = processor.FuturesExecutor(workers=nworkers, merging=(1, 30, 10000))
```

Comment:
What does this do?

Reply:
it sets up how merging steps N chunks requirement. It's something like requires at least 30 chunks to have finished, with max chunk merging size of 10000 and need to have at least a one of those to fire a merge.

It's just a parameter that determines when to start merging.

Reply:
Interesting, how big is the effect and how did you optimize it? (What is the runtime difference with default vs this optimized version?)

Reply:
not really big... it didn't matter that much when I tried a few. I did not optimize it systematically. but i didn't want to merge every 5 files, which was the default setting.